readme.txt

```diff
@@ -4,7 +4,7 @@ Donate link: https://wpgraphql.com/acf
 Tags: WPGraphQL, GraphQL, API, Advanced Custom Fields, ACF
 Requires at least: 5.0
 Tested up to: 5.1.1
-Stable tag: 0.4.1
+Stable tag: 0.5.0

```

Comment:
Since this is significant breaking changes, should you put something in the readme description or add an  'upgrade notice' section? See https://wordpress.org/plugins/readme.txt

Reply:
Since the plugin isn't distributed on WordPress.org (yet) I've been handling all my communication in Github releases. 

Once we hit 1.0 and distribute on the .org repo, we'll use this README much more actively like I'm doing for WPGraphQL 

This shouldn't actually cause breaking changes either. It's for sure possible, but the tests we did have show that the location rules that used to work still work, and added more coverage for additional location rules. 

I bumped it from 0.4.1 to 0.5.0 because it is significant changes, but should still be non-breaking.